tabs/src/components/app/views/SampleQueries.jsx

```diff
@@ -54,9 +60,19 @@ export function FetchSamples(props) {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    return (
-        <Table rows={samples} aria-label="sample queries" />
-    );
+    if (isLoading) {
+        return (
+            <Loader
+                size='medium'
+                label={t("Sample Queries.loading")}
+            />
+        );
+    } else {
+        return (
+            <Table rows={samples} aria-label="sample queries" />
+        );
+    }
+
```

Comment:
can we use ternary statements? the project has been using mostly ternary statements